### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by tolwi** [tolwi/hassio-ecoflow-cloud#321] [Announcements]
# Initial release with public API support

## What's Changed

* Initial RIVER_MINI support by @Bravo13 in https://github.com/tolwi/hassio-ecoflow-cloud/pull/263
* Fixed `AC In Volts` sensor for River Pro by @merlin-zaraza in https://github.com/tolwi/hassio-ecoflow-cloud/pull/259
* public api by @tolwi in https://github.com/tolwi/hassio-ecoflow-cloud/pull/265
* River Max: Add "Auto Fan Speed" switch by @kasper334 in https://github.com/tolwi/hassio-ecoflow-cloud/pull/267
* Fixed backup reserve enabled for delta 2 and other devices by @SergiyUKR in https://github.com/tolwi/hassio-ecoflow-cloud/pull/269
* Corrections + Regroupement des appareils sur une Entrée by @Angelius007 in https://github.com/tolwi/hassio-ecoflow-cloud/pull/281
* Add ukrainian by @AndriiZakhliupanyi in https://github.com/tolwi/hassio-ecoflow-cloud/pull/313
* Fix a little typo in powerstream.py by @Dev-Fabry in https://github.com/tolwi/hassio-ecoflow-cloud/pull/317
* 300-add-esp-temperature by @Dev-Fabry in https://github.com/tolwi/hassio-ecoflow-cloud/pull/318

## New Contributors
* @Bravo13 made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/263
* @merlin-zaraza made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/259
* @kasper334 made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/267
* @SergiyUKR made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/269
* @Angelius007 made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/281
* @AndriiZakhliupanyi made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/313
* @Dev-Fabry made their first contribution in https://github.com/tolwi/hassio-ecoflow-cloud/pull/317

**Full Changelog**: https://github.com/tolwi/hassio-ecoflow-cloud/compare/v0.13.4...v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/tolwi/hassio-ecoflow-cloud/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by bswan** [comment]
Public API seems to be working for me, got api key from here: https://developer-eu.ecoflow.com/ and able to connect to api and get list of my devices, sadly public api implementation does not support my devices at the moment :D Please add River Pro and River 2 Pro. Maybe I can help with implementation somehow?

**Reply by merlin-zaraza** [reply]
That's the question to Ecoflow team. I don't think we can help. You can still use old private API with your devices

**Reply by bswan** [reply]
Mmmm, I don't think I worded it right :) Let me illustrate :)
See, I can connect to the API and here are all my devices:
![image](https://github.com/user-attachments/assets/a4957d69-e33f-4ae6-8cdf-3a02d70db568)

But if I select any of them, next menu brings slight dissapointment:
![image](https://github.com/user-attachments/assets/1dbc4c6d-654a-46ad-877e-e30d194cd5ad)

So I need River Pro and River 2 Pro on that second list :) 

For now I added them as "Diagnostic" Will diagnostic output help to speed this up? :)

**Reply by merlin-zaraza** [reply]
You can use application API, and then choose your device:
![image](https://github.com/user-attachments/assets/8e2273cb-005d-4581-9a10-2b62d1e4f8ba)

**Reply by bswan** [reply]
Yeah, I know, mate, I have them setup this way now, but we got public API now so we don't have to rely on "hacked" way :) So let's bring my devices on to this API too :)

**Reply by bswan** [reply]
So how can I help with that? :)
